### DISCUSSION TITLE
运行脚本时老是出现user root: parse error at 7错误

### DISCUSSION THREAD

**Discussion by aliyy2021** [whyour/qinglong#1799] [Q&A] [answered]
龙大，你好！
运行脚本时老是出现user root: parse error at 7错误，查了好久，找不到问题所在，重装过，配置文件也反复确认了，实在找不到原因，请大佬指点，谢谢 
![image](https://user-images.githubusercontent.com/77948441/217439510-ed6d73bd-cb86-464d-8124-ac4640ecab6b.png)

**Comment by aliyy2021** [comment, marked as answer]
问题已解决，不知道咋删除，请大佬删除

**Comment by cjh19831115** [comment]
如何解决的，分享一下

**Reply by aliyy2021** [reply]
有些定时文件格式有问题，打开crontab.list文件仔细看看。

**Reply by OwnerCM** [reply]
> 有些定时文件格式有问题，打开crontab.list文件仔细看看。

大佬能不能更详细的说明一下怎么检查的，我一行行检查发现都没问题

**Comment by shuwey** [comment]
配置中的“DefaultCronRule”的值是不是为空，如果是就填入值即可（如：0 9 * * *）
